static/src/javascripts/projects/common/modules/commercial/creatives/hosted-thrasher.js

```diff
@@ -0,0 +1,41 @@
+define([
```

Comment:
Doesn't this need to be bundled into breakout-iframe before it'll work outside of dev?